src/effects/VST3/VST3Wrapper.cpp

```diff
@@ -564,7 +681,7 @@ void VST3Wrapper::LoadPreset(const wxString& presetId)
    {
       throw SimpleMessageBoxException(
          ExceptionType::Internal,
-         XO("Unable to apply VST3 preset file %s").Format(presetId),
+         XO("Unable to apply VST3 preset %s").Format(presetId),
```

Comment:
This is a change of a user facing string that I suppose you must tell design about, if there is any important opinion about string consistency.

Reply:
Right, thanks!

Reply:
Reverted that line